clippy_book/book.toml

```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["Moaz bin Mohamed Mokhtar"]
```

Comment:
Putting yourself as the sole author when this builds on a lot of stuff that has been there does feel inappropriate here.

Reply:
It is inappropriate at all which is not fair of-course.

I'm a beginner here; what the general manners to follow regarding that point?
Should I put all in this RP?
Should the reviewer or admin fix it before publishing to add who participated in this task?

Reply:
> what the general manners to follow regarding that point?

> Should the reviewer or admin fix it before publishing to add who participated in this task?

We never talked about this in Clippy before. We should talk about it in our meeting tomorrow. All the other crates in Clippy are also currently attributed to specific authors.

> Should I put all in this RP?

No this PR should only focus on the book. PRs usually address only one issue.

Reply:
We just decided that it should read "The Rust Clippy Developers", and there will be another PR to change this everywhere else.

Reply:
That means I will change authors to the "Cargo.toml" in main directory. It is listed as below:
```rust
authors = [
	"Manish Goregaokar <manishsmail@gmail.com>",
	"Andre Bogus <bogusandre@gmail.com>",
	"Georg Brandl <georg@python.org>",
	"Martin Carton <cartonmartin@gmail.com>",
	"Oliver Schneider <clippy-iethah7aipeen8neex1a@oli-obk.de>"
]
```

Reply:
No this was already done in #6878. You want to rebase on top of master to get those changes though.